pubpub/pubpub
# generalize the GDPR banner to show other banners as well

Here we press the GDPR banner into service for sequentially showing other legal notifications as well — in this case, for a terms of service update. When users accept the TOS change, we store a `tos-update` cookie with the current date and then use that to make sure they don't see it again (on this device).

I have made some styling changes to this banner so it has `position: fixed` and lives at the bottom of the screen. I think this is a little better than living at the top and reflowing the page content, but I have almost no opinion about this and am happy to defer to others.

Managing this banner will be done with a code change for the time being — overall easier than mucking with environment variables or whatnot.

Closes #681 

_Test plan:_
- Open a new browser session and make sure you see the GDPR banner, and that accepting or denying the third-party cookies causes the TOS banner to show.
- Click the "See terms" button and verify that it opens a link to the TOS in a new window.
- Close the banner. On the next page reload, you shouldn't see it, or the GDPR banner, again.

_Screenshot:_
![image](https://user-images.githubusercontent.com/2208769/73193907-52e51280-40f9-11ea-8276-cd66737962bf.png)
